neptune/new/run.py

```diff
@@ -141,397 +93,18 @@ def __init__(
-        return self._get_subpath_suggestions()
-
-    @assure_run_not_stopped
-    def __getitem__(self, path: str) -> 'Handler':
-        return Handler(self, path)
-
-    @assure_run_not_stopped
-    def __setitem__(self, key: str, value) -> None:
-        self.__getitem__(key).assign(value)
-
-    @assure_run_not_stopped
-    def __delitem__(self, path) -> None:
-        self.pop(path)
-
-    @assure_run_not_stopped
-    def assign(self, value, wait: bool = False) -> None:
-        """Assign values to multiple fields from a dictionary.
-        You can use this method to quickly log all run's parameters.
-
-        Args:
-            value (dict): A dictionary with values to assign, where keys become the paths of the fields.
-                The dictionary can be nested - in such case the path will be a combination of all keys.
-            wait (bool, optional): If `True` the client will first wait to send all tracked metadata to the server.
-                This makes the call synchronous. Defaults to `True`.
-
-        Examples:
-            >>> import neptune.new as neptune
-            >>> run = neptune.init()
-
-            >>> # Assign multiple fields from a dictionary
-            ... params = {"max_epochs": 10, "optimizer": "Adam"}
-            >>> run["parameters"] = params
-
-            >>> # You can always log explicitely parameters one by one
-            ... run["parameters/max_epochs"] = 10
-            >>> run["parameters/optimizer"] = "Adam"
-
-            >>> # Dictionaries can be nested
-            ... params = {"train": {"max_epochs": 10}}
-            >>> run["parameters"] = params
-            >>> # This will log 10 under path "parameters/train/max_epochs"
-
-        You may also want to check `assign docs page`_.
-
-        .. _assign docs page:
-            https://docs.neptune.ai/api-reference/run#assign
-        """
-        self._get_root_handler().assign(value, wait)
-
-    @assure_run_not_stopped
-    def fetch(self) -> dict:
-        """Fetch values of all non-File Atom fields as a dictionary.
-        The result will preserve the hierarchical structure of the run's metadata, but will contain only non-File Atom
-        fields.
-        You can use this method to quickly retrieve previous run's parameters.
-
-        Returns:
-            `dict` containing all non-File Atom fields values.
-
-        Examples:
-            >>> import neptune.new as neptune
-            >>> resumed_run = neptune.init(run="HEL-3")
-            >>> params = resumed_run['model/parameters'].fetch()
-
-            >>> run_data = resumed_run.fetch()
-
-            >>> print(run_data)
-            >>> # this will print out all Atom attributes stored in run as a dict
-
-        You may also want to check `fetch docs page`_.
-
-        .. _fetch docs page:
-            https://docs.neptune.ai/api-reference/run#fetch
-        """
-        return self._get_root_handler().fetch()
-
     def ping(self):
```

Comment:
It should be moved to Container as well, shouldn't it?

Reply:
We thought (with @Herudaio) that a generic `Container` doesn't need to be pinged -- I was under the impression that `Project` won't have an `active` parameter (and no `running_time`, I think?), so pinging doesn't make sense.

Reply:
discussed offline -- `Project` **can** have `active` and `running_time` , so moving `ping` to `AttributeContainer`